mathics/builtin/box/image.py

```diff
@@ -27,4 +30,12 @@ def boxes_to_mathml(self, elements=None, **options):
         )
 
     def boxes_to_tex(self, elements=None, **options):
-        return "-Image-"
+        fp = tempfile.NamedTemporaryFile(delete=True, suffix=".png")
```

Comment:
Consider using pillow routines for writing PNG.

Reply:
The conversion was done when Image is converted in ImageBox. imageBox stores a B64 encoded version of the PNG file. Here we just decode it and store it in a file.

Reply:
If ImageBox can keep the pillow structure, that may be a win.  A more general problem we have is that in digesting things for M-Expressions we lose the efficient and sometimes more flexible properties of whatever the object was before.  And we spend a lot of time in conversion.

Reply:
This is done now.